openedx/features/caliper_tracking/tests/expected/edx.course.student_notes.used_unit_link.json

```diff
@@ -0,0 +1,41 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "action": "NavigatedTo",
+  "actor": {
+    "id": "http://localhost:18000/u/edx",
+    "name": "edx",
```

Comment:
Please use a name other than "edx". This is easily confused with `org_id`

Reply:
done